src/App.js

```diff
@@ -4,8 +4,12 @@ function App() {
   return (
     <div className="App">
       <header className="App-header">My cakes</header>
-      <main className="App-main">Content</main>
+      <main className="App-main"></main>
       <footer className="App-footer">
+        <li className="Item-card">
+          <img src="" alt="" />
+          <h3 className="Item-card__name">Chocolate cake</h3>
```

Comment:
nice usage of h-tag. i think h3 is very good for that usage. and BEM is also great.